File: core/src/main/java/net/minestom/server/utils/math/IntRange.java
package net.minestom.server.utils.math;

public class IntRange extends Range<Integer> {

    public IntRange(Integer minimum, Integer maximum) {
        super(minimum, maximum);
    }

    public IntRange(Integer value) {
        super(value);
    }

    /**
     * {@inheritDoc}
     */
    @Override
    public boolean isInRange(Integer value) {
        return value >= this.getMinimum() && value <= this.getMaximum();
    }
}

File: src/main/java/net/minestom/server/utils/math/IntRange.kt
package net.minestom.server.utils.math

class IntRange : Range<Int> {
    constructor(minimum: Int, maximum: Int) : super(minimum, maximum) {}
    constructor(value: Int) : super(value) {}

    /**
     * {@inheritDoc}
     */
    override fun isInRange(value: Int): Boolean {
        return value >= minimum && value <= maximum
    }
}